modules/iasBidAdapter.js

```diff
@@ -103,6 +111,16 @@ function interpretResponse(serverResponse, request) {
   shallowMerge(commonBidResponse, getPageLevelKeywords(iasResponse));
   commonBidResponse.slots = iasResponse.slots;
   bidResponses.push(commonBidResponse);
+
+  otherBidIds.forEach(function (bidId) {
+    var otherResponse = Object.assign({}, commonBidResponse);
+    otherResponse.requestId = bidId;
+    bidResponses.push(otherResponse);
+  });
+
+  if (top.postIASResponse) {
```

Comment:
This has the possibility of throwing an exception if within an unfriendly iFrame.  Should wrap with try/catch

Reply:
looking (comment to make sure this PR stays open)